webpack.config.js

```diff
@@ -29,80 +29,16 @@ env.setBabelEnv();
 const pathResolver = new PathResolver();
 const globber = new Globber({pathResolver});
 const pluginFactory = new PluginFactory({globber});
-const copyrightBannerPlugin = pluginFactory.createCopyrightBannerPlugin();
 const cssBundleFactory = new CssBundleFactory({env, pathResolver, globber, pluginFactory});
 const jsBundleFactory = new JsBundleFactory({env, pathResolver, globber, pluginFactory});
 
 const OUTPUT = {
   fsDirAbsolutePath: pathResolver.getAbsolutePath('./build'),
-  httpDirAbsolutePath: '/assets/',
 };
 
-if (env.isDev()) {
```

Comment:
FYI this got lost in translation and I suspect this may be the real cause of what we attempted to address in #2437. I think we should revert that and re-integrate this check.

Currently in master, `isDev()` is never checked whatsoever, #2437 causes `npm run build`  to unconditionally always build both demo files and dist files, and the fact that we patched the `build` target rather than the webpack configs means this regression slipped by `test:buildconfig` unnoticed.